src/cwv/handler.js

```diff
@@ -44,7 +45,69 @@ export async function CWVRunner(auditUrl, context, site) {
   };
 }
 
+export async function convertToOppty(auditUrl, auditData, context) {
+  const {
+    dataAccess,
+    log,
+  } = context;
+
+  const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  let opportunity = opportunities.find((oppty) => oppty.getType() === AUDIT_TYPE);
+
+  if (!opportunity) {
+    const opportunityData = {
+      siteId: auditData.siteId,
+      auditId: auditData.id,
+      runbook: 'https://adobe.sharepoint.com/sites/aemsites-engineering/Shared%20Documents/3%20-%20Experience%20Success/SpaceCat/Runbooks/Experience_Success_Studio_CWV_CLS_Runbook.docx?web=1',
+      type: AUDIT_TYPE,
+      origin: 'AUTOMATION',
+      title: 'Core Web Vitals',
+      description: 'Core Web Vitals are key metrics Google uses to evaluate website performance, impacting SEO rankings by measuring user experience.',
+      guidance: {
+        steps: [
+          'Analyze CWV data using RUM and PageSpeed Insights to identify performance bottlenecks.',
+          'Optimize CWV metrics (CLS, INP, LCP) by addressing common issues such as slow server response times, unoptimized assets, excessive JavaScript, and layout instability.',
+          'Test the implemented changes with tools like Chrome DevTools or PageSpeed Insights to verify improvements.',
+          'Monitor performance over time to ensure consistent CWV scores across devices.',
+        ],
+      },
+      tags: [
+        'SEO',
+        'Performance Optimization',
+        'User Experience',
+        'Core Web Vitals',
+        'Organic Traffic',
+        'Web Performance',
+      ],
+    };
+    opportunity = await dataAccess.Opportunity.create(opportunityData);
+  } else {
+    opportunity.setAuditId(auditData.id);
+    await opportunity.save();
+  }
+
+  // Sync suggestions
+  const buildKey = (data) => (data.type === 'url' ? data.url : data.pattern);
+
+  await syncSuggestions({
+    opportunity,
+    newData: auditData.auditResult.cwv,
+    buildKey,
+    mapNewSuggestion: (entry) => ({
+      opportunityId: opportunity.getId(),
+      type: 'CODE_CHANGE',
+      rank: entry.pageviews,
```

Comment:
@ekremney This rank property will be used to decide sort order of issues on the UI.

For cwv, I haven't seen any discussion on the sort order in the UI. Hence, checking with you if you already know that this is the correct way to check the impact, or do we need to check with Hyman on this?
I am thinking it could be somehow decided on the basis of both pageviews and cwv poor/needs-improvement

For each opportunity, we are checking the impact as it fits there e.g. this is how we do it for meta tags:
https://github.com/adobe/spacecat-audit-worker/pull/494/files#diff-9e627bac711d499c0db20db00eb8e64bc847a7bfd51d3355f8c8a11d2fff3429R113

Reply:
Started a discussion on slack here:
https://cq-dev.slack.com/archives/C05A45JBP9N/p1733129944880019

Reply:
here is the final impl: the back-end returns rank fields sorted by type and organic. Group types have higher ranks then URLs. Each group are sorted by the organic traffic in themselves